### DISCUSSION TITLE
【求助】有个问题

### DISCUSSION THREAD

**Discussion by JobNumber-9527** [share121/inter-knot#2562] [其他]
就是我在其他人帖子底下评论后，底下再有回复就算不是回复我的，也会发到我的QQ邮箱，怎么解决啊<img src="https://img-static.mihoyo.com/communityweb/upload/ee7bb800b0c90f66fa9e321cb6d8ee17.png" alt="抱大腿" width="75" >

**Comment by HifumiMizuhara** [comment]
点右边的Unsubscribe
<img width="265" alt="スクリーンショット 2024-08-17 19 27 31" src="https://github.com/user-attachments/assets/a2a7daea-349b-4f56-abae-42c3f05ea11c">

**Reply by JobNumber-9527** [reply]
谢谢<img src="https://img-static.mihoyo.com/communityweb/upload/fe198bb1523528c2b852d28626379a2a.png" alt="心情简单" width="75" >

**Comment by 00zzxx** [comment]
就楼上老哥发的那个，然后未读通知里面可以一次性退订之前的所有

**Reply by 00zzxx** [reply]
以后发完记得多点一下就OK了
